cookieconsent/admin.py

```diff
@@ -0,0 +1,3 @@
+from django.contrib import admin
+
+# Register your models here.
```

Comment:
Delete these empty files or keep them?

Reply:
Good to delete the empty files if not used.